### ISSUE TITLE
Secret from google authenticator doesn't work

### ISSUE THREAD

**Issue by PaulVelthuis93** [ejisan/scalauthx-otp#2]
This application works perfect when its generates its own secrets. However I tried a secret from googleAuthenticator and got the following

java.lang.NumberFormatException: Illegal embedded sign character

The secret I used was: CMVW2EQFHEFF4FS6

Hopefully it is possible to guide me where the error is.

**Comment by ejisan** [comment]
Can I have stack trace and your code how you use it that helps a lot to debug it.
Thank you 😃

**Comment by PaulVelthuis93** [comment]
then I pasted the secret to test your script.  
val secret = "CMVW2EQFHEFF4FS6"
val secretbase = OTPSecretKey.fromBase32(secret)
println(secretbase)

This results in the following:
at java.math.BigInteger.<init>(BigInteger.java:428)
at scala.math.BigInt$.apply(BigInt.scala:83)
at ejisan.scalauthx.otp.Base32$.decode(Base32.scala:22)
at ejisan.scalauthx.otp.OTPSecretKey$.fromBase32(OTPSecretKey.scala:57)

When I do the following it does work
val secretbase = OTPSecretKey.fromBase32("CMVW2EQFHEFF4FS6")
println(secretbase)

I found the error it was wrongly formatted my String (UTF-8) that's why it didn't recognise. This topic can be closed.
